docker-compose.yml

```diff
@@ -60,20 +60,23 @@ volumes:
 services:
   database:
     image: timescale/timescaledb:latest-pg15
+    hostname: database
     environment:
       - TZ=Europe/Berlin
       - POSTGRES_HOST=database
       - POSTGRES_PORT=5432
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=postgres
+    command: postgres -c 'max_connections=200'
     volumes:
       - data-postgres:/var/lib/postgresql/data:delegated
     networks:
       net-mgmt:
         ipv4_address: 192.168.10.3
 
   horizon:
-    image: opennms/horizon:latest
+    image: opennms/horizon:32.0.1
```

Comment:
@indigo423 I think it's best we stick to fixed tags for the OpenNMS components in this repo rather than latest. We can bump up when we confirm something works.